getting-started/installation-apt-debian.md

```diff
@@ -2,7 +2,9 @@
 
 This will install TimescaleDB via `apt` on Debian distros.
 
-**Note: TimescaleDB requires PostgreSQL 9.6.3+, 10.9+, or 11.4+**
+**Note: TimescaleDB requires PostgreSQL 11.4 or later. Support for
+PostgreSQL 9.6.3+ and 10.9+ is deprecated and may not be supported in
+a future release.**
```

Comment:
This reads a bit like "support for foo... is not supported". I suggest tweaking:

```suggestion
**Note: TimescaleDB requires PostgreSQL 11.4 or later. Support for
PostgreSQL 9.6.3+ and 10.9+ is deprecated and may be removed in
a future release.**
```

This applies also to the other places where this occurs.